test_data/user_defined_structs/expected_representation.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Placeholder_1 do
+  defstruct [:wheels, :doors, :color]
```

Comment:
Eventually, we will want these user defined keys to be replaced as mentionned in #37, right?

Reply:
Yes